integration-test/src/main/java/com/sequenceiq/it/cloudbreak/testcase/authorization/CreateDhWithDatahubCreator.java

```diff
@@ -82,18 +87,21 @@ public void testCreateEnvironmentWithDh(TestContext testContext) {
                 .await(EnvironmentStatus.AVAILABLE)
                 // testing unauthorized calls for environment
                 .whenException(environmentTestClient.describe(), ForbiddenException.class, expectedMessage("Doesn't have 'environments/describeEnvironment'" +
-                        " right on 'environment' " + environmentShortPattern(testContext)).withWho(Actor.useRealUmsUser(AuthUserKeys.ENV_CREATOR_B)))
+                        " right on 'environment' " + environmentShortPattern(testContext)).withWho(cloudbreakActor.useRealUmsUser(AuthUserKeys.ENV_CREATOR_B)))
                 .whenException(environmentTestClient.describe(), ForbiddenException.class, expectedMessage("Doesn't have 'environments/describeEnvironment'" +
-                        " right on 'environment' " + environmentShortPattern(testContext)).withWho(Actor.useRealUmsUser(AuthUserKeys.ENV_CREATOR_B)))
+                        " right on 'environment' " + environmentShortPattern(testContext)).withWho(cloudbreakActor.useRealUmsUser(AuthUserKeys.ZERO_RIGHTS)))
```

Comment:
did you change the rights intentionally?

Reply:
Yes. Unfortunately this was changed mistakenly by my previous [1c635b55a7b86ab312166a8d14e9fe25a29f76c9](https://github.com/hortonworks/cloudbreak/commit/1c635b55a7b86ab312166a8d14e9fe25a29f76c9#diff-a9ac0b63b70ca4d2b930b820ebe436a37685f6c9ba84f5cdf4b2c16690ac334f) commit. 

Original values are:
https://github.com/hortonworks/cloudbreak/blob/2a14ab75d9ebefc7727b96bd24917ad9210679fc/integration-test/src/main/java/com/sequenceiq/it/cloudbreak/testcase/authorization/CreateDhWithDatahubCreator.java#L88-L92